src/components/curriculum/__snapshots__/CurriculumPartTime.test.js.snap

```diff
@@ -29,78 +29,92 @@ exports[`<CurriculumPartTime /> should render component 1`] = `
         name="session1"
         title="Session 1 - Modern JavaScript"
         toggleNavigateTo={[Function]}
+        type="Part-time"
       >
         <ES6Session />
       </CurriculumSection>
       <CurriculumSection
         name="session2"
         title="Session 2 - Thinking in React"
         toggleNavigateTo={[Function]}
+        type="Part-time"
       >
         <ThinkingInReactSession />
       </CurriculumSection>
       <CurriculumSection
         name="session3"
         title="Session 3 - Routing & Data Fetching"
         toggleNavigateTo={[Function]}
+        type="Part-time"
       >
         <RoutingAndDataFetchingSession />
       </CurriculumSection>
       <CurriculumSection
         name="session4"
         title="Session 4 - Forms & Auth"
         toggleNavigateTo={[Function]}
+        type="Part-time"
       >
         <FormsAndAuthSession />
       </CurriculumSection>
       <CurriculumSection
         name="session5"
         title="Session 5 - Recap React Fundamentals"
         toggleNavigateTo={[Function]}
+        type="Part-time"
       >
         <ReactFundamentalsRecapSession />
       </CurriculumSection>
-    </Styled(Col)>
-    <Styled(Col)
-      lg={5}
-      md={6}
-    >
       <CurriculumSection
         name="session6"
         title="Session 6 - Styling in React"
         toggleNavigateTo={[Function]}
+        type="Part-time"
       >
         <StylingInReactSession />
       </CurriculumSection>
       <CurriculumSection
         name="session7"
         title="Session 7 - Introduction to Redux"
         toggleNavigateTo={[Function]}
+        type="Part-time"
       >
         <IntroReduxSession />
       </CurriculumSection>
       <CurriculumSection
         name="session8"
         title="Session 8 - Introduction to Testing in JS"
         toggleNavigateTo={[Function]}
+        type="Part-time"
       >
         <TestingIntroSession />
       </CurriculumSection>
       <CurriculumSection
         name="session9"
         title="Session 9 - Advanced Patterns I"
         toggleNavigateTo={[Function]}
+        type="Part-time"
```

Comment:
Why we are not using here the variables in `data.js` ??

Reply:
@horacioh we are. This is the output of the snapshot test so that is the string that the variable resolves to ;-)